spark/interpreter/src/main/java/org/apache/zeppelin/spark/SparkSqlInterpreter.java

```diff
@@ -115,14 +115,23 @@ public InterpreterResult internalInterpret(String st, InterpreterContext context
           context.out.flush();
           return new InterpreterResult(Code.ERROR);
         } else {
+          LOGGER.error("Error happens in sql: " + curSql, e);
```

Comment:
Please use the parameterized method.

Reply:
The last parameter is Exception, I think we can not use parameterized method here.

Reply:
You are right.

Reply:
Hi @zjffdu,
I was a little bit confused. It is possible to use parameterised methods for exceptions as well.
Checkout: http://www.slf4j.org/faq.html#paramException

Reply:
You are right, it is supported, I verified it locally, learned new things :)

Reply:
I also learned a lot from the Zeppelin project.